kogito-usertasks-with-security-oidc-keycloak/src/main/resources/META-INF/resources/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>kogito-infinispan-persistence-quarkus - 1.0-SNAPSHOT</title>
```

Comment:
I guess we could remove this file completely ...

Reply:
sure